packages/amplify-cli-core/src/cliViewAPI.ts

```diff
@@ -0,0 +1,67 @@
+//Use this file to store all types used between the CLI commands and the view/display functions
+// CLI=>(command-handler)==[CLI-View-API]=>(ux-handler/report-handler)=>output-stream
+import chalk from 'chalk';
+export interface CLIParams {
+    cliCommand : string;
+    cliSubcommands: string[]|undefined;
+    cliOptions : {[key:string] : any};
```

Comment:
This is definitely required. I can make this change as part of a subsequent PR, since the current code has some refactoring on resource-status.ts ( yet to be pushed ) besides the original feature in the PR. 
The problem with cliOptions is that it's  derived from `context.input.options`. In our code, we currently have context as any (this too will be solidified in subsequent PRs). Based on its usage in the code, it seems to be a generically defined type ( with the intent to pass any object from the command-line handler to the CLI engine. ) 
Hence it would be mitigate some risks by moving this change to the next PR